src/Symfony/Component/Validator/Constraints/Bic.php

```diff
@@ -38,14 +41,25 @@ class Bic extends Constraint
     );
 
     public $message = 'This is not a valid Business Identifier Code (BIC).';
+    public $ibanMessage = 'This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.';
+    public $iban;
+    public $ibanPropertyPath;
 
-    public function __construct($options = null)
+    public function __construct(array $options = null)
     {
         if (!class_exists(Intl::class)) {
             // throw new LogicException(sprintf('The "symfony/intl" component is required to use the "%s" constraint.', __CLASS__));
             @trigger_error(sprintf('Using the "%s" constraint without the "symfony/intl" component installed is deprecated since Symfony 4.2.', __CLASS__), E_USER_DEPRECATED);
         }
 
+        if (isset($options['iban']) && isset($options['ibanPropertyPath'])) {
+            throw new ConstraintDefinitionException(sprintf('The "iban" and "ibanPropertyPath" of the Iban constraint cannot be used at the same time.', \get_class($this)));
+        }
+
+        if (isset($options['propertyPath']) && !class_exists(PropertyAccess::class)) {
+            throw new LogicException(sprintf('The "symfony/property-access" component is required to use the "%s" constraint with the "ibanPropertyPath" option.', \get_class($this)));
```

Comment:
`with the "propertyPath" option`

Reply:
@ro0NL but the option really is `ibanPropertyPath`!